webui/src/views/OkrEditor.vue

```diff
@@ -3,7 +3,8 @@
         <header class="header">
             <h2 class="title">OKR Portal</h2>
 
-            <span class="user-name">{{user.name}}</span>
+            <span class="user-name" @click="logOut">Sign Out</span>
```

Comment:
@preigile may be we can move "Sign Out" into translatable resources little bit later

Reply:
Yes, of course, and all other texts

Reply:
Please, give this class another name, for example `header-nav-item`